plugin/vite-configs/location/web.ts.timestamp-1668988989654.mjs

```diff
@@ -0,0 +1,93 @@
+// vite-configs/location/web.ts
+import { defineConfig } from "vite";
```

Comment:
What is this web.ts.timestamp-23423423423.mjs file? If this was output from your build that is wrong. If this file was created on purpose, please let me know what its for.

Reply:
it is a temporary file, it shouldn't be existed , it is created during build : web , actually it is not in my env. now

Reply:
sorry I found it in my env  but it is temporary , I removed it